### DISCUSSION TITLE
Introducing the DreamBerd Vision Pro

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#580] [Announcements]
Today we're proud to announce the DreamBerd Vision Pro.

It's **not** AR or VR. It's *spatial computing*. And Lu Wilson has done a FULL COMPREHENSIVE VIDEO REVIEW that's completely unbiased and unsponsored.

Check it out [here](https://youtu.be/QRKnrFEjDF0).

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/vpro'>Introducing the DreamBerd Vision Pro</a>.</em>

**Comment by NotAShelf** [comment]
I envision that **DreamBerd Vision Pro** will change our lives forever. Already invested.

(unsponsored comment)

**Comment by Colon101** [comment]
the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro the dreamberd vision pro

**Comment by ishyv** [comment]
**DreamBerd Vision Pro** was indeed the **DreamBerd Vision Pro**

**Comment by RedstoneWizard08** [comment]
Thank you, @TodePond, you just made my day.
